tests/e2e/trustyai_test.go

```diff
@@ -75,3 +83,67 @@ func (c *TrustyAITestCtx) disableKserve(t *testing.T) {
 		BeEmpty(),
 	)
 }
+
+func (c *TrustyAITestCtx) validateCRDReinstated(t *testing.T) {
+	// validate multikuueclusters, multikueueconfigs has v1beta1 version
+
+	crds := []string{
+		"inferenceservices.serving.kserve.io", // SR is not needed any more in 2.18.0
+	}
+
+	for _, crd := range crds {
+		t.Run(crd, func(t *testing.T) {
+			c.ValidateCRDReinstated(t, crd)
+		})
+	}
+}
+
+func (c *TrustyAITestCtx) validateTrustyAIPreCheck(t *testing.T) {
+	// validate precheck on CRD version:
+	// pre-req: skip trusty to removed (done by ValidateComponentDisabled)
+	// step: delete isvc left from enableKserve wait till it is gone
+	// set trustyai to managed, result to error, enable mm, result to success
```

Comment:
Why are we enabling ModelMeshServing when we should re-enable back Kserve which deployed isvc originally?

Reply:
so we can have better coverage